src/style.scss

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin: 0;
+  margin-top: 20px;
+}
+
+.cell {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). Fix all cases please